tests/bluetooth/bsim_bt/bsim_test_adv_resume/test_scripts/run_test.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
```

Comment:
Since this shell scrip will be run by the `run_parallel.sh`, it would be better to give it a more descriptive name, especially since we now have multiple `run_test.sh` scripts.